pkgs/top-level/all-packages.nix

```diff
@@ -1933,6 +1933,8 @@ with pkgs;
     postgresql = postgresql_14;
   };
 
+  nose2pytest = with python3Packages; toPythonApplication nose2pytest;
```

Comment:
Shouldn't we use `pkgs/by-name` for this?

Reply:
It’s also a library; its output can depend on itself for complex cases. (Which is probably not that useful, but hey.)

Or are you saying that I should put a stub in `pkgs/by-name` that just calls `toPythonApplication` on the library? Does anything else do that?

Reply:
> Or are you saying that I should put a stub in pkgs/by-name that just calls toPythonApplication on the library?

Yes precisely.

Reply:
Well… sure, why not. Pushed.

Reply:
In most cases, we simply add a line to toplevel/all-packages.nix, but that is ok.